scene/gui/color_picker.cpp

```diff
@@ -127,7 +132,6 @@ void fragment() {
 	COLOR = vec4(clamp((abs(fract(((a - TAU) / TAU) + vec3(3.0, 2.0, 1.0) / 3.0) * 6.0 - 3.0) - 1.0), 0.0, 1.0), (b + b2 + b3 + b4) / 4.00);
 }
 )");
-
```

Comment:
What happened to this line? 🙃 (looks like accidental removal)

Reply:
What was removed? 
![image](https://user-images.githubusercontent.com/32321/161862264-ebaa028c-6e21-45d0-a78f-501867746843.png)

Reply:
If it's an empty line, it's ok to remove or not remove. It does take like 10m to 30 m to run cicd though.

Reply:
Well, you could restore it if you do any other changes here. That line separated shaders (you even added such line for the 3rd shader).

Reply:
Updated.